packages/@react-aria/overlays/src/usePopover.ts

```diff
@@ -96,7 +96,7 @@ export function usePopover(props: AriaPopoverProps, state: OverlayTriggerState):
     isDisabled: isNonModal
   });
 
-  useLayoutEffect(() => {
+  useEffect(() => {
```

Comment:
I think this was intentionally changed to be a useLayoutEffect in a prior PR? Digging to see why and if it is safe to change this to a useEffect

Reply:
From @devongovett https://github.com/adobe/react-spectrum/pull/3677/files#diff-02c0eb0f2cdf8d726564712562f36c693aa5d9feb5860837da0a88f3eac01a51R98-R102, I think he's taking a look too.